src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -547,3 +547,16 @@ g.test('reverse_depth')
       }
     );
   });
+
+g.test('create_pipeline_with_all_formats')
```

Comment:
I wrote this test referring to `DepthStencilStateTest.CreatePipelineWithAllFormats` test[1]. But, I'm not fully sure if adding this test is useful. Please let me know if this test is useless here. 

[1] https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/tests/end2end/DepthStencilStateTests.cpp;l=763?q=depthstencil&ss=chromium

Reply:
It's useful on its own, but it doesn't do much since it doesn't actually use the pipeline, just creates it. I'd prefer to have coverage of this through tests that actually test rendering. `depth.spec.ts` already has a test with cases over all formats with a depth aspect. If we can parameterize one or more similar tests in `stencil.spec.ts` to all formats with a stencil aspect - which we should have anyway - then that'll be much stronger than this test.

Reply:
ok, then it's likely it would be good to make the `stencil_compare_func` test on other stencil texture formats like the `depth_compare_func` test in depth.spec.ts.